python/setup.py

```diff
@@ -218,7 +218,7 @@ def get_packages(self):
         ],
         "serve": ["uvicorn==0.16.0", "requests", "starlette", "fastapi", "aiorwlock"],
         "tune": ["pandas", "tabulate", "tensorboardX>=1.9", "requests"],
-        "k8s": ["kubernetes", "urllib3"],
+        "k8s": ["kubernetes==23.3.0", "urllib3"],
```

Comment:
I'm not an expert in these things, but one inequality seems like it could allow installation of an old incompatible thing and the other could allow a future breakage.

Reply:
Also, double equals is consistent with what we do for the other "cloud provider" modules.

Reply:
Hmm, I guess you can do a range for these...

Reply:
Actually, yeah <=23.3.0 should be fine.

Reply:
Actually, I prefer to pin it exactly until someone complains.

Reply:
I'll just remove the pin.

Reply:
^ This is a hotfix to the ray operator, will consider ramifications of pinning another time.